applications/ShapeOptimizationApplication/python_scripts/algorithm_trust_region.py

```diff
@@ -0,0 +1,682 @@
+# ==============================================================================
+#  KratosShapeOptimizationApplication
+#
+#  License:         BSD License
+#                   license: ShapeOptimizationApplication/license.txt
+#
+#  Main authors:    Baumgaertner Daniel, https://github.com/dbaumgaertner
+#
+# ==============================================================================
+
+# Making KratosMultiphysics backward compatible with python 2.6 and 2.7
+from __future__ import print_function, absolute_import, division
+
+# Kratos Core and Apps
+from KratosMultiphysics import *
+from KratosMultiphysics.ShapeOptimizationApplication import *
```

Comment:
e.g. `import KratosMultiphysics.ShapeOptimizationApplication as KratosShapeOpt` would be better